src/test/java/nextstep/optional/ComputerStoreTest.java

```diff
@@ -42,18 +43,20 @@ public void getVersionOptional() {
 
     @Test
     public void getVersionOptionalWhenComputerIsNull() throws Exception {
-        assertThat(ComputerStore.getVersionOptional(null)).isEqualTo(ComputerStore.UNKNOWN_VERSION);
+        assertThat(ComputerStore.getVersionOptional(null)).isEqualTo(UNKNOWN_VERSION);
     }
 
     @Test
     public void getVersionOptionalWhenSoundcardIsNull() throws Exception {
         Computer computer = new Computer(null);
-        assertThat(ComputerStore.getVersionOptional(computer)).isEqualTo(ComputerStore.UNKNOWN_VERSION);
+        String s = ComputerStore.getVersionOptional(computer);
+        System.out.println("s = " + s);
```

Comment:
디버깅용으로 사용했던 코드는 삭제해 주세요.
테스트 코드라도 변수명을 축약하지 않으면 더 좋을 것 같아요.
```suggestion
        String version = ComputerStore.getVersionOptional(computer);

```